modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,17 @@
+.jetpack-simple-payments__purchase-message {
+	display: none;
+	color: #fff;
```

Comment:
Can we move only this color rule to a different block with a selector with more specificity so that it remains in white in most themes?

```
body .jetpack-simple-payments__wrapper p.jetpack-simple-payments__purchase-message {
color: #fff;
}
```